hedera-node/hedera-network-service-impl/src/test/java/com/hedera/node/app/service/network/impl/test/serdes/MonoSpecialFilesAdapterSerdesTest.java

```diff
@@ -45,19 +45,13 @@ class MonoSpecialFilesAdapterSerdesTest {
         SOME_SPECIAL_FILES.append(PbjConverter.fromPbj(SOME_FILE_ID), "abcdef".getBytes());
     }
 
-    @Mock
-    private ReadableSequentialData input;
-
-    @Mock
-    private WritableSequentialData output;
-
     final MonoSpecialFilesAdapterCodec subject = new MonoSpecialFilesAdapterCodec();
 
     @Test
     void doesntSupportUnnecessary() {
         assertThrows(UnsupportedOperationException.class, () -> subject.measureRecord(SOME_SPECIAL_FILES));
-        assertThrows(UnsupportedOperationException.class, () -> subject.measure(input));
-        assertThrows(UnsupportedOperationException.class, () -> subject.fastEquals(SOME_SPECIAL_FILES, input));
+        assertThrows(UnsupportedOperationException.class, () -> subject.measure(null));
```

Comment:
This also changes the effect. If the intent is to ensure these methods always throw, then perhaps calling with both null and non-null inputs would be advisable.